### PR TITLE
Add configurable timeout for gunicorn workers

### PR DESCRIPTION
**Pull request by Winnie-Fred**
I added `timeout` setting to gunicorn config which is configurable via environment variable.

### CODE REVIEW

File: src/config/gunicorn.py [hunk 1]
```diff
@@ -12,3 +12,5 @@
 threads = int(os.getenv("PYTHON_MAX_THREADS", 1))
 
 reload = bool(strtobool(os.getenv("WEB_RELOAD", "false")))
+
+timeout = int(os.getenv("GUNICORN_TIMEOUT", 120))
```
Comment (nickjj) [inline src/config/gunicorn.py:16]: Can you rename this to `WEB_TIMEOUT`?

File: .env.example [hunk 2]
```diff
@@ -70,6 +70,10 @@ export WEB_CONCURRENCY=1
 #export WEB_RELOAD=false
 export WEB_RELOAD=true
 
+# GUNICORN_TIMEOUT sets the timeout for gunicorn workers, defaults to 120
```
Comment (nickjj) [inline .env.example:73]: `# Configure the timeout value in seconds for gunicorn.`

**Review by nickjj — CHANGES_REQUESTED** [review]
Hi, thanks for the PR.

File: .env.example [hunk 3]
```diff
@@ -70,9 +70,8 @@ export WEB_CONCURRENCY=1
 #export WEB_RELOAD=false
 export WEB_RELOAD=true
 
-# GUNICORN_TIMEOUT sets the timeout for gunicorn workers, defaults to 120 
-# seconds.
-#export GUNICORN_TIMEOUT=
+# Configure the timeout value in seconds for gunicorn.
+#export WEB_TIMEOUT=
```
Comment (nickjj) [inline .env.example:74]: Given it defaults to 120, can you also set `export WEB_TIMEOUT=120` here so folks know what the default value is?

**Review by nickjj — CHANGES_REQUESTED** [review]

File: .env.example (hunk 3 above)
Comment (Winnie-Fred) [reply .env.example:74]: Sure thing.

**Comment by nickjj** [comment]
Looks good, can you roll up the commits to a single one with a message of "Add configurable gunicorn timeout value"?

**Comment by Winnie-Fred** [comment]
Done, I have rolled up the commits.

**Review by nickjj — APPROVED** [review]
Looks good, thanks a lot for the PR.

**Pull request merged**